app/models/support_interface/vendor_api_monitor.rb

```diff
@@ -1,66 +1,39 @@
 module SupportInterface
   class VendorAPIMonitor
+    def providers_with_api_usage_stats
```

Comment:
The following query is being run every time for the different sections, with the added scoping depending on what data we need. However we do not need all the left joins every time. Could we instead move the relevant joins to the sections that require them to avoid querying them unnecessarily, this could help with the `where.not` issue for `last_syncs` and `last_decisions`. Let me know your thoughts